### PR TITLE
ui: data quality improvements

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes:

I worked on comments

- https://github.com/open-metadata/OpenMetadata/issues/10543#issuecomment-1518013655
- https://github.com/open-metadata/OpenMetadata/issues/10543#issuecomment-1518925238

- [x] Fixed the error thrown on the test case details page
- [x] Horizontal scrollbar visible in test case table
- [x] Too much space between `Parameter` and values
- [x] Padding in the chart is not proper.
- [x] Test case details page is broken.
- [x] The parameter values should be displayed vertically

https://user-images.githubusercontent.com/51777795/233841465-a0ab3111-f41e-4b63-87a5-b446d24dafe5.mov

- [x] Changing the datePicker default value to last3days

https://user-images.githubusercontent.com/51777795/233880331-261b9e9a-7075-4d18-b873-585bd6a2680f.mov

#
### Type of change:

- [x] Bug fix
- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/styles/components/table.less [hunk 1]
```diff
@@ -62,6 +62,11 @@
             border-bottom: none;
           }
         }
+        // Removed padding for expanded rows to prevent unnecessary scrolling behavior for the expanded row content
+        tbody > .ant-table-expanded-row > td {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/styles/components/table.less:66]: This looks risky, can we keep it to a specific table for now? will add it globally in 1.1

Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/styles/components/table.less:66]: Sure, I'll make the chhanges.

**Comment by harshach** [comment]
@aniketkatkar97 please change the default time period to last 3 days not yesterday.

**Comment by aniketkatkar97** [comment]
@harshach I have updated it already. The first video is before those changes. I've uploaded a second video to show the `last3days` changes

**Review by harshach — APPROVED** [review]

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**